bucket/enpass.json

```diff
@@ -1,13 +1,31 @@
 {
     "homepage": "https://www.enpass.io/",
-    "version": "6.0.4.281",
-    "url": "https://dl.enpass.io/stable/windows/package/EnpassPackage-6.0.4.281.zip",
-    "hash": "36d3f0041700fbeb55f62c9c05cbb8c4cf825af022a355a668bf9202b22a66eb",
+    "version": "6.1.0.407",
+    "url": "https://dl.enpass.io/stable/windows/package/EnpassPackage-6.1.0.407.zip",
+    "hash": "dbc3c7e814314fdd2e98e52c46cce0d1a936cf3223d7bf13086b584c1f181d47",
     "bin": "Enpass.exe",
     "installer": {
         "script": [
-            "Expand-7zipArchive \"$dir\\Enpass*setup.exe\" \"$dir\"",
-            "Remove-Item \"$dir\\Enpass*setup.exe\""
+            "Expand-DarkArchive \"$dir\\Enpass-setup.exe\" \"$dir\\tmp\"",
+            "Expand-MsiArchive \"$dir\\tmp\\AttachedContainer\\Enpass.msi\" \"$dir\\tmp\" | Out-Null",
+            "movedir \"$dir\\tmp\\PFiles\\Enpass\" \"$dir\" | Out-Null"
+        ]
+    },
+    "post_install": [
```

Comment:
I do not like forcing registry manipulation for users. 

That is the reason why I use scoop. To ditch installers bad behaviour of setting registry.

See vscode, tortoisesvn manifests how to implement this optionally.

Reply:
I can't say I disagree with a fair amount of conservatism on dumping stuff in the registry ;) 

In this case however I see no other way to fix this. 

Enpass has cloud sync capabilities as core functionality, for which it needs browser interaction to setup authentication.
AFAIK these handlers are the only way to get that working without things going confusingly wrong (they're the smallest subset of reg entries I could make).
Notice I also properly remove the keys as necessary, and they don't seem likely to interfere elsewhere.

So making it optional would only make installation more of a hassle and errorprone to the vast majority of the userbase.

Fortunately it's not like we're dumping 3 tons of uncleanable garbage in there for no apparent reason, unlike some apps we know.

Reply:
@Ash258 bump?